assets/xml/objects/object_firefly.xml

```diff
@@ -1,54 +1,81 @@
-        <Limb Name="object_firefly_Standardlimb_001834" Type="Standard" EnumName="OBJECT_FIREFLY_LIMB_1A" Offset="0x1834" />
-        <Limb Name="object_firefly_Standardlimb_001840" Type="Standard" EnumName="OBJECT_FIREFLY_LIMB_1B" Offset="0x1840" />
-        <Skeleton Name="object_firefly_Skel_0018B8" Type="Normal" LimbType="Standard" LimbNone="OBJECT_FIREFLY_LIMB_NONE" LimbMax="OBJECT_FIREFLY_LIMB_MAX" EnumName="ObjectFireflyLimb" Offset="0x18B8" />
+        <!-- Assets for the Fire Keese-->
+
+        <!-- Fire Keese Flying Animation-->
+        <Animation Name="gFireKeeseFlyAnim" Offset="0x17C" /> <!-- Original name is "firefly_foot_fly" -->
+
+        <!--Fire Keese Textures-->
+        <Texture Name="gFireKeeseBodyTex" OutName="fire_keese_body" Format="rgba16" Width="8" Height="8" Offset="0x210" />
+        <Texture Name="gFireKeeseEyeTex" OutName="fire_keese_eye" Format="rgba16" Width="8" Height="8" Offset="0x290" />
+        <Texture Name="gFireKeeseEarTex" OutName="fire_keese_ear" Format="rgba16" Width="16" Height="8" Offset="0x310" />
+        <Texture Name="gFireKeeseTalonTex" OutName="fire_keese_talon" Format="rgba16" Width="8" Height="16" Offset="0x410" />
+        <Texture Name="gFireKeeseWingTex" OutName="fire_keese_wing" Format="rgba16" Width="4" Height="16" Offset="0x510" />
+
+        <!--Regular Keese Textures-->
+        <Texture Name="gKeeseRedEyeTex" OutName="keese_red_eye" Format="rgba16" Width="8" Height="8" Offset="0x190" />
+        <Texture Name="gKeeseBodyTex" OutName="keese_body" Format="rgba16" Width="8" Height="8" Offset="0x590" />
+        <Texture Name="gKeeseTalonTex" OutName="keese_talon" Format="rgba16" Width="8" Height="16" Offset="0x610" />
+        <Texture Name="gKeeseWingTex" OutName="keese_wing" Format="rgba16" Width="4" Height="16" Offset="0x710" />
+        <Texture Name="gKeeseEarTex" OutName="keese_ear" Format="rgba16" Width="16" Height="8" Offset="0x790" />
+
+        <!--DLists-->
+        <DList Name="gFireKeeseHeadDL" Offset="0xD30" />
+        <DList Name="gFireKeeseBodyDL" Offset="0xF40" />
+        <DList Name="gFireKeeseLeftFootDL" Offset="0x1038" />
+        <DList Name="gFireKeeseRightFootDL" Offset="0x1100" />
+        <DList Name="gFireKeeseRightWingStartDL" Offset="0x11C8" />
+        <DList Name="gFireKeeseRightWingMidDL" Offset="0x1290" />
+        <DList Name="gFireKeeseRightWingEndDL" Offset="0x1358" />
+        <DList Name="gFireKeeseLeftWingEndDL" Offset="0x1420" />
+        <DList Name="gFireKeeseLeftWingMidDL" Offset="0x14E8" />
+        <DList Name="gFireKeeseLeftWingStartDL" Offset="0x15B0" />
+        <DList Name="gKeeseRedEyesDL" Offset="0x1678" />
+
+        <!--Limbs for the whole thing-->
+        <Limb Name="gFireKeeseWholeRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_WHOLE_ROOT" Offset="0x1708" />
+        <Limb Name="gFireKeeseWholeWrapperLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_WHOLE_WRAPPER" Offset="0x1714" />
```

Comment:
```suggestion
        <Limb Name="gFireKeeseRootLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_ROOT" Offset="0x1708" />
        <Limb Name="gFireKeeseRootWrapperLimb" Type="Standard" EnumName="FIRE_KEESE_LIMB_ROOT_WRAPPER" Offset="0x1714" />
```
When it is just the `RootLimb` is generally implied for the whole thing, so the word WHOLE isn't necessary imo.